src/Machine.js

```diff
@@ -573,6 +578,20 @@ class Machine {
     console.error("\x1b[33m" + message + '\u001b[39m');
   }
 
+  /**
+   * Execute "info" instruction
+   * @param {{type, value}} instruction
+   * @param {{}} context
+   * @param {string[]} buffer
+   * @private
+   */
+     _executeInfo(instruction, context, buffer) {
+      const message = this.expression.evaluate(instruction.value,
+        context
+      );
+      console.log("\x1b[33m" + message + '\u001b[39m');
```

Comment:
What color does this emit the log as?

Reply:
It'll be yellow since I copied it from the warning one and forgot to change, should I just remove the formatting or use some other color?

Reply:
I'd suggest blue (I think that's somewhat typical for info)

Reply:
Done!